src/images/mixer-icon-outline-white.svg

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
what is this? the image?

Reply:
All the <g></g> that is

Reply:
Yeah svg's are comprised of code the make up the image